scripts/ci/linkTaps.js

```diff
@@ -0,0 +1,56 @@
+const { exec } = require('child_process');
```

Comment:
This file along with the `scripts/ci/linkRepo.sh` file makes it overly complicated.

Why not just do everything in one file?
  * You could either call the `tap link` task directly from this file
  *  Or loop over the taps to link in the `scripts/ci/linkRepo.sh` file

In my opinion, it would be better to search the `keg-hub/repos/` sub folders
  * Within each sub-folder, check if it has a tap.js(on) file
  * If it does, then update the global config
    * By calling the `tap link` task
    * Or by calling `keg-cli/src/utils/globalConfig/addTapLink`
      * This is the method  the `tap link` task calls to update the global config

Reply:
Yeah all good points. 

I updated it to use one single script (`linkRepos.sh`).

That script loops over each repo in `keg-hub/repos`, and then it attempts to link that repo.

If that repo doesn't have a tap-config alias key, then it skips it. 

So we won't need to update this script in the future when we add more injected taps; it should just work.